### DISCUSSION TITLE
EthereumStratum failed...is that a problem?

### DISCUSSION THREAD

**Discussion by hmercker** [catchthatrabbit/coreminer#44] [Miner]
When starting up the mining process with bash ./mine.sh, below message come sup. Could that reduce any performance/functionality of the mining process?

coreminer 0.19.0+commit.0607dbad
Build: linux/release/gnu

cp 11:31:40 main     RandomY hard aes mode is enabled
cp 11:31:40 main     RandomY full mem mode is enabled
cp 11:31:40 main     RandomY JIT mode is enabled
 i 11:31:43 main     Configured pool eu.catchthatrabbit.com:8008
 i 11:31:43 main     Configured pool eu1.catchthatrabbit.com:8008
 i 11:31:43 <unknown> Selected pool eu.catchthatrabbit.com:8008
 i 11:31:44 <unknown> Negotiation of EthereumStratum/2.0.0 failed. Trying another ...
 i 11:31:44 <unknown> Negotiation of EthereumStratum/1.0.0 failed. Trying another ...
 i 11:31:45 <unknown> Stratum mode : Eth-Proxy compatible
 i 11:31:45 <unknown> Established connection to eu.catchthatrabbit.com [86.110.248.193:8008]
 i 11:31:45 <unknown> Spinning up miners...
cp 11:31:45 <unknown> Using CPU: 0 Memory : 35.97 GB
cp 11:31:45 <unknown> Using CPU: 1 Memory : 35.97 GB
 i 11:31:45 <unknown> Epoch : -1 Difficulty : 470.27 Kh
 i 11:31:45 <unknown> Job: 7df3ef8c… eu.catchthatrabbit.com [86.110.248.193:8008]
 m 11:31:48 <unknown> 0:00 A0 0.00 h - cp0 0.00, cp1 0.00

**Comment by rastislavcore** [comment]
All is fine, no issues.
